src/components/sbb-header/sbb-header.stories.js

```diff
@@ -113,6 +118,14 @@ WithUserMenu.argTypes = basicArgTypes;
 WithUserMenu.args = { ...basicArgs };
 WithUserMenu.play = playStory;
 
```

Comment:
we probably should have a story or e2e test where there is a different scrollorigin than the document. Then we can see if all is working.